src/test/ui/fn/fn-fat-arrow-return.rs

```diff
@@ -0,0 +1,4 @@
+fn g() => usize { 0 }
```

Comment:
Could you add a test for multiple `=>` ?

```rust
fn foo() => Fn() => bool {
    unimplemented!()
}
```

Reply:
And also `=>` in arguments.
```rust
fn foo(bar: impl Fn() => bool) {
    unimplemented!()
}
```

Reply:
I don't think either of these could be supported without hitting the same ambiguity as described in [my other comment](https://github.com/rust-lang/rust/pull/77035#discussion_r492621515). Worth checking, but that's my expectation.